tests/integration/parity/common.py

```diff
@@ -30,6 +30,10 @@ def _check_web3_clientVersion(self, client_version):
 
 
 class ParityEthModuleTest(EthModuleTest):
+    def test_eth_getBlockByNumber_pending(self, web3):
+        pytest.xfail('Parity deprecated "pending" option in 1.11.1')
```

Comment:
I think this comment should say they removed (or dropped) the option instead of deprecating it. (If I understand correctly)

If it's just deprecated, I would expect the call to continue working.